src/assets/scss/_reset.scss

```diff
@@ -0,0 +1,112 @@
+@import "./mixin";
+* {
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+@font-face {
+  font-family: 'ArialMT';
+  src: url('../fonts/ArialMTBlack.woff2') format('woff2'),
+  url('../fonts/ArialMTBlack.woff') format('woff'),
+  url('../fonts/ArialMTBlack.ttf') format('truetype');
+  @include font_face_design;
+}
+body {
+  margin: 0;
+  color: var(--body-text-color);
+  -webkit-text-size-adjust:  $w_100;
+  -ms-text-size-adjust: none;
+  font-size: 16px;

```

Comment:
write a single line syntax for font-size and font family with line-height.

Reply:
I have added a single line instead of multiple lines... just like this (font: size/lineheight family;)